### PR TITLE
add com.github.franjegueje.deckico

### PR DESCRIPTION
**Pull request by FranjeGueje**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.franjegueje.deckico.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - type: file
+        path: deckico-gui
+      - type: file
+        path: deckico-cli
+      - type: file
+        path: yad
+      - type: dir
+        path: .
+      - type: dir
+        path: ./images
```
Comment (JakobDev) [inline com.github.franjegueje.deckico.yml:32]: This will not work

File: com.github.franjegueje.deckico.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+app-id: com.github.franjegueje.deckico
```
Comment (hfiguiere) [inline com.github.franjegueje.deckico.yml:1]: this should be `io.github`

File: com.github.franjegueje.deckico.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: deckico-gui
+finish-args:
+  - --socket=x11
+  - --filesystem=~/.config/autostart
+  - --filesystem=~/.local/share/Steam/steamapps/compatdata
+  - --filesystem=~/.local/share/Steam/steamapps/shadercache
+  - --filesystem=~/.local/share/Steam/userdata:ro
```
Comment (hfiguiere) [inline com.github.franjegueje.deckico.yml:11]: this will not work for a flatpak installed steam.

File: com.github.franjegueje.deckico.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      - install -D yad /app/bin/yad
+      - mkdir -p /app/usr/share/icons/deckico
+      - cp -Rf images/* /app/usr/share/icons/deckico/.
+    sources:
+      - type: file
+        path: deckico-gui
+      - type: file
+        path: deckico-cli
+      - type: file
+        path: yad
```
Comment (hfiguiere) [inline com.github.franjegueje.deckico.yml:28]: 1. these files are not found
2. what they seem to look like is that they wouldn't belong here.

You should either build from source (preferred) which mean pulling upstream sources here, or use a prebuilt binary.

File: com.github.franjegueje.deckico.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      - type: dir
```
Comment (hfiguiere) [inline com.github.franjegueje.deckico.yml:29]: `dir` types should be avoided.

File: com.github.franjegueje.deckico.yml (hunk 1 above)
Comment (hfiguiere) [inline com.github.franjegueje.deckico.yml:32]: these are missing too. But they should instead come from upstream. See comment above.

File: com.github.franjegueje.deckico.yml (hunk 3 above)
Comment (FranjeGueje) [reply com.github.franjegueje.deckico.yml]: This flatpak will be used to Steam on local (Steam Deck) not for flatpak installed steam

File: com.github.franjegueje.deckico.yml (hunk 4 above)
Comment (FranjeGueje) [reply com.github.franjegueje.deckico.yml]: I have added the files

File: com.github.franjegueje.deckico.yml (hunk 5 above)
Comment (FranjeGueje) [reply com.github.franjegueje.deckico.yml]: I removed references to "dir". I now use tar.gz

File: com.github.franjegueje.deckico.yml (hunk 1 above)
Comment (FranjeGueje) [reply com.github.franjegueje.deckico.yml]: I have added the files and deleted the dirs

File: io.github.franjegueje.deckico.yml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  - --filesystem=~/.local/share/Steam/userdata:ro
+  - --filesystem=/run/media
+modules:
+  - name: deckico
+    buildsystem: simple
+    build-commands:
+      - install -D deckico-gui /app/bin/deckico-gui
+      - install -D deckico-cli /app/bin/deckico-cli
+      - install -D yad /app/bin/yad
+      - mkdir -p /app/usr/share/icons/deckico
```
Comment (hfiguiere) [inline io.github.franjegueje.deckico.yml:20]: this path is incorrect. it is NEVER `/app/usr`

File: io.github.franjegueje.deckico.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      - tar -xvzf images.tar.gz -C /app/usr/share/icons/deckico/.
+    sources:
+      - type: file
+        path: deckico-gui
+      - type: file
+        path: deckico-cli
```
Comment (hfiguiere) [inline io.github.franjegueje.deckico.yml:26]: these are part of "upstream" therefor they don't belong here, as I previously said.

File: io.github.franjegueje.deckico.yml [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+      - type: file
+        path: yad
```
Comment (hfiguiere) [inline io.github.franjegueje.deckico.yml:28]: this is a binary. Same applies as above.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
Flathub repositories are meant to handle the manifest to build a flatpak. not to hold binary build or artifact from upstream project.

File: io.github.franjegueje.deckico.yml (hunk 8 above)
Comment (hfiguiere) [reply io.github.franjegueje.deckico.yml:28]: also how is this gonna work with the aarch64 package built?

**Pull request closed without merging**